Explain OES_standard_derivatives functions (#35710)
mdn/content: files/en-us/web/api/oes_standard_derivatives/index.md

genType fwidth(genType)
```

- `dFdx()`
  - : Returns the derivative in `x` using local differencing for the input argument `p`.
- `dFdy()`

>   - : Returns the derivative in `y` using local differencing for the input argument `p`. These two functions are commonly used to estimate the filter width used to anti-alias procedural textures.

Reviewer: Unfortunate that "These two functions" is glued to this one function. It might be better to put this after or before the `<dl>`? Or is this just what `fwidth()` is?

- `fwidth()`

>   - : Returns the sum of the absolute derivative in `x` and `y` using local differencing for the input argument `p`. I.e. `abs(dFdx(p)) + abs(dFdy(p))`

Reviewer: ```suggestion
  - : Returns the sum of the absolute derivative in `x` and `y` using local differencing for the input argument `p`. That is:`abs(dFdx(p)) + abs(dFdy(p))`.
```
